dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -54,21 +49,21 @@ def __init__(self, plugin_dest_dir = None):
         #   major version and the minor version in the prebuilt plugin is lower than the requested one.
         self.can_install_prebuilt = not self.always_build and \
             bool(self.tf_compiler) and \
-            self.tf_compiler in self.prebuilt_compilers and \
+            (StrictVersion(self.default_cpp_version) >= StrictVersion('5.0') and \
```

Comment:
Why you check the system compiler here? We don't need that to check prebuild. Only the second check is needed (regarding TF compiler version).